index.html

```diff
@@ -34,5 +34,131 @@
       </li>
     </ul>
     </div>
+    <div id="recent">
+      <div id="title"><p>My Recent Works</p></div>
+      <span id="line"><hr/></span>
+    <div id="frame">
+        <div id="caption_img"> <img src="./Image/Img Placeholder.png"/></div>
+        <div id="content">
+        <div id="stories"><p>Multi-Post Stories</p></div>
+        <div id="summary">A daily selection of privately personalized reads, no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500's, when an unknown printer took a standard dummy text.</div>
+    <div id="bar">
+        <div id="css">css</div>
+        <div class=""="html">html</div>
+        <div class=""="boot">bootstrap</div>
+        <div class=""="ruby">Ruby</div>
```

Comment:
I have noticed that the languages in you design have been placed and positioned inside one bar. But when i look at the template each of these languages are placed in their own seperate box. Again kindly inspect your template properly.

This is your design:
![Screenshot from 2022-07-02 16-11-07](https://user-images.githubusercontent.com/79506105/177000664-e8b8fa1d-70b8-4fe7-9597-a7bd4388e0f8.png)
 
Please stick to this Figma design:

![Screenshot from 2022-07-02 16-26-02](https://user-images.githubusercontent.com/79506105/177000738-14f58287-7974-4028-b319-d3126dbaa744.png)